examples/01_plotting/plot_demo_glass_brain_extensive.py

```diff
@@ -2,48 +2,75 @@
 Glass brain plotting in nilearn (all options)
 =============================================
 
-This example goes through different options of the :func:`nilearn.plotting.plot_glass_brain` function
-(including plotting negative values).
+First part of this example goes through different options of the
+:func:`nilearn.plotting.plot_glass_brain` function (including plotting
+negative values).
 
-See :ref:`plotting` for more plotting functionalities.
+Second part, goes through same options but selected of the same glass brain
+function but plotting is seen with contours.
+
+See :ref:`plotting` for more plotting functionalities and Section 4.3. for
```

Comment:
I think that it would be nice to have the text "Section 4.3." to be a link to that section.